Packs/EWS/Integrations/EWSO365/EWSO365.py

```diff
@@ -192,7 +192,7 @@ def __init__(
         )
         self.folder_name = folder
         self.is_public_folder = is_public_folder
-        self.access_type = kwargs.get('access_type') or IMPERSONATION
+        self.access_type = kwargs.get('access_type', '').lower() or IMPERSONATION
```

Comment:
for instances that were configured prior updating this, will there be `None` as the value of that param ?

Reply:
Yeah